pkg/kubelet/cm/pod_container_manager_linux.go

```diff
@@ -73,6 +75,10 @@ func (m *podContainerManagerImpl) EnsureExists(pod *v1.Pod) error {
 	// check if container already exist
 	alreadyExists := m.Exists(pod)
 	if !alreadyExists {
+		enforceCPULimits := m.enforceCPULimits
+		if utilfeature.DefaultFeatureGate.Enabled(kubefeatures.DisableCPUQuotaWithExclusiveCPUs) && m.podContainerManager.PodHasExclusiveCPUs(pod) {
```

Comment:
let's add a log entry here

Reply:
added! (v=2)

Reply:
do we need to log the value of `unified` here?